rpm/systemd-240-test-fix-tests-for-supplementary-groups.patch

```diff
@@ -0,0 +1,152 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Yu Watanabe <watanabe.yu+github@gmail.com>
+Date: Sat, 29 Sep 2018 21:05:52 +0900
+Subject: [PATCH] test: fix tests for supplementary groups
+
+Fixes #9881.
+---
+ ...xec-dynamicuser-fixeduser-one-supplementarygroup.service | 3 ++-
+ test/test-execute/exec-dynamicuser-fixeduser.service        | 3 ++-
+ .../exec-dynamicuser-supplementarygroups.service            | 4 +++-
+ ...mentarygroups-multiple-groups-default-group-user.service | 6 +++++-
+ ...exec-supplementarygroups-multiple-groups-withgid.service | 5 ++++-
+ ...exec-supplementarygroups-multiple-groups-withuid.service | 4 +++-
+ .../exec-supplementarygroups-single-group-user.service      | 3 ++-
+ .../exec-supplementarygroups-single-group.service           | 3 ++-
+ test/test-execute/exec-supplementarygroups.service          | 3 ++-
+ 9 files changed, 25 insertions(+), 9 deletions(-)
+
+diff --git a/test/test-execute/exec-dynamicuser-fixeduser-one-supplementarygroup.service b/test/test-execute/exec-dynamicuser-fixeduser-one-supplementarygroup.service
+index de1a6e7303..bd7881d286 100644
+--- a/test/test-execute/exec-dynamicuser-fixeduser-one-supplementarygroup.service
++++ b/test/test-execute/exec-dynamicuser-fixeduser-one-supplementarygroup.service
+@@ -2,7 +2,8 @@
+ Description=Test DynamicUser with User= and SupplementaryGroups=
+ 
+ [Service]
+-ExecStart=/bin/sh -x -c 'test "$$(id -G)" = "1" && test "$$(id -g)" = "1" && test "$$(id -u)" = "1"'
++ExecStart=/bin/sh -x -c 'HAVE=0; for g in $$(id -G); do test "$$g" = "1" && HAVE=1; done; test "$$HAVE" -eq 1'
```

Comment:
id -G | grep -E '\<1\>' comes to mind as slightly simpler way to do this, but as this is upstream patch... nevermind